src/components/Facility/DischargedPatientsList.tsx

```diff
@@ -75,6 +155,40 @@ const DischargedPatientsList = ({
     }
   }, [qParams]);
 
+  const date_range_fields = [
+    [params.created_date_before, params.created_date_after],
+    [params.modified_date_before, params.modified_date_after],
+    [params.date_declared_positive_before, params.date_declared_positive_after],
+    [params.last_vaccinated_date_before, params.last_vaccinated_date_after],
+    [
+      params.last_consultation_encounter_date_before,
+      params.last_consultation_encounter_date_after,
+    ],
+    [
+      params.last_consultation_discharge_date_before,
+      params.last_consultation_discharge_date_after,
+    ],
+  ];
+
+  const calculateDateRangeDuration = (
+    startDate: string,
+    endDate: string,
+  ): number => {
+    if (!startDate && !endDate) return 0;
+    if (startDate && endDate) {
+      return Math.abs(dayjs(endDate).diff(dayjs(startDate), "days"));
+    }
+    return -1; // Invalid range when only one date is set
+  };
```

Comment:
consider moving this to a utility function. let's not duplicate implementations.

Reply:
Moved to utils.js and also updated the managePateints.tsx to use ulity function